### PR TITLE
Dashboard Integration for test_pvc_clone_performance

### PR DESCRIPTION
**Pull request by pintojoy**
In this PR test_pvc_clone_performance_results are pushed into ES server and dashboard. Also base class is changed from E2Etest to PASTest

### CODE REVIEW

File: tests/e2e/performance/csi_tests/test_pvc_clone_performance.py [hunk 1]
```diff
@@ -203,4 +251,80 @@ def test_clone_create_delete_performance(
...
+        and reporting the full results (links in the ES) of previous tests (4)
+        """
+        self.number_of_tests = 8
+        self.results_path = get_full_test_logs_path(
+            cname=self, fname="test_clone_create_delete_performance"
+        )
+        self.results_file = os.path.join(self.results_path, "all_results.txt")
+        logger.info(f"Check results in {self.results_file}.")
+        self.check_tests_results()
+        self.push_to_dashboard(test_name="pvc clone performance")
```
Comment (ypersky1980) [inline tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:314]: I'd name the test : "PVC Clone Performance"  with Caps lock - similar to the convention of other tables names.

**Review by ypersky1980 — DISMISSED** [review]

File: tests/e2e/performance/csi_tests/test_pvc_clone_performance.py (hunk 1 above)
Comment (pintojoy) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py]: Thanks Yuli. I made that change

**Review by ypersky1980 — APPROVED** [review]
LGTM, thank you!

File: tests/e2e/performance/csi_tests/test_pvc_clone_performance.py [hunk 2]
```diff
@@ -203,4 +251,80 @@ def test_clone_create_delete_performance(
...
+            full_results (obj): an FIOResultsAnalyse object
+
+        Returns:
+            FIOResultsAnalyse (obj): the input object fill with data
+
+        """
+        for key in self.environment:
+            full_results.add_key(key, self.environment[key])
+        full_results.add_key("index", full_results.new_index)
+        return full_results
```
Comment (ramkiperiy) [inline tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: Can this function be moved to generic library file?

Comment (ramkiperiy) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: @Avilir @ypersky1980 @pintojoy ^^ if its more specific to test-case then ok with it but all the duplicate function do have same lines of codes.

Comment (ypersky1980) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: @pintojoy @ramkiperiy I think that we have to have this def init_full_results(self, full_results) in each one of our tests - this is initialization of the results obj.

Comment (ypersky1980) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: we do have a generic class ResultsAnalyse(PerfResult): in ocs-ci/ocs_ci/ocs/perfresult.py /

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/perfresult.py#L163

Comment (ramkiperiy) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: @ypersky1980 yes, even my point was same why can't we add this function `init_full_results` in the class `class ResultsAnalyse(PerfResult):` and call it in each test script ? 
@Avilir WDYT about this ?

Comment (Avilir) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: @ramkiperiy , the reason for this object is to hendale the results, calculating average , summarize results etc.

**Review by Avilir — APPROVED** [review]
LGTM

File: tests/e2e/performance/csi_tests/test_pvc_clone_performance.py (hunk 2 above)
Comment (ramkiperiy) [reply tests/e2e/performance/csi_tests/test_pvc_clone_performance.py:330]: As discussed in the meeting, let's have this tracked part of code optimization and reducing code duplicates.

**Review by ramkiperiy — APPROVED** [review]
